tomcat.spec

```diff
@@ -121,4 +120,22 @@ if [ "$1" == 2 ]; then
     fi
 fi
 
+#Conditions for installing the manager and host-manager applications
```

Comment:
Huh, based on the output in the build it looks like it's trying to run this during the construction of the RPM not during install.